Dockerfile

```diff
@@ -14,17 +28,13 @@ RUN true \
     && ln -s /usr/share/wazo-setupd/50-wazo-plugin-nestbox.yml /etc/wazo-webhookd/conf.d/50-wazo-plugin-nestbox.yml \
     && install -d -o wazo-setupd -g wazo-setupd /run/wazo-setupd/ \
     && install -o wazo-setupd -g wazo-setupd /dev/null /var/log/wazo-setupd.log \
-    && apt-get -yqq autoremove \
     && openssl req -x509 -newkey rsa:4096 -keyout /usr/share/xivo-certs/server.key -out /usr/share/xivo-certs/server.crt -nodes -config /usr/share/xivo-certs/openssl.cfg -days 3650 \
     && chown wazo-setupd:wazo-setupd /usr/share/xivo-certs/*
 
-COPY . /usr/src/wazo-setupd
-WORKDIR /usr/src/wazo-setupd
-RUN true \
-  && pip install -r /usr/src/wazo-setupd/requirements.txt \
-  && python setup.py install \
-  && cp -r etc/* /etc
-
 EXPOSE 9302
 
-CMD ["python3", "-u", "/usr/local/bin/wazo-setupd"]
+ENV PYTHONUNBUFFERED=TRUE
```

Comment:
Why is this necessary?

Reply:
I personally, never experiment this behavior but it's to avoid to hide some log by the buffer when a crash occurs. I think seb has already debug this and it take some time to understand why... so it's why we have this option activated in some services